src/Symfony/Component/Serializer/Serializer.php

```diff
@@ -179,7 +172,7 @@ public function normalize($data, string $format = null, array $context = [])
             throw new NotNormalizableValueException(sprintf('Could not normalize object of type "%s", no supporting normalizer found.', get_debug_type($data)));
         }
 
-        throw new NotNormalizableValueException('An unexpected value could not be normalized: '.(!\is_resource($data) ? var_export($data, true) : sprintf('%s resource', get_resource_type($data))));
+        throw new NotNormalizableValueException('An unexpected value could not be normalized: '.(!\is_resource($data) ? var_export($data, true) : sprintf('"%s" resource', get_resource_type($data))));
```

Comment:
This is an unrelated change and should be reverted.

Reply:
This was required by fabbot.it